### PR TITLE
Removing overview stepper unwanted new tab launch buttons

### PR DESCRIPTION
**Pull request by CrowleyRajapakse**
## Issues
Fixes https://github.com/wso2/product-apim/issues/10807

## Screenshot

Screenshot 2021-04-10 at 12 32 53

### CODE REVIEW

**Comment by Krishanx92** [comment]
Few concerns regarding this
-- Color of the Link should not change(Purple indicate visited links), we need to handle it
-- User should aware of these links (May be add a new icon)

@erangatl

**Review by erangatl — COMMENTED** [review]
Links are hard to identify, shall we use the link icons for internal links
image

**Comment by Krishanx92** [comment]
> Links are hard to identify, shall we use the link icons for internal links
> image

+1  and keep text as it is.

**Comment by CrowleyRajapakse** [comment]
Fixed as follows
Screenshot 2021-04-10 at 15 30 41

**Review by erangatl — APPROVED** [review]

**Review by Krishanx92 — APPROVED** [review]

**Comment by tmkasun** [comment]
image

here text and the icon are not aligned?

**Comment by tmkasun** [comment]
Shall we make the text itself clickable ?

**Comment by CrowleyRajapakse** [comment]
Fixed as below
Screenshot 2021-04-10 at 22 02 58

Screenshot 2021-04-10 at 22 03 22

**Comment by CrowleyRajapakse** [comment]
changed the link word color to black
Screenshot 2021-04-10 at 22 52 06

Screenshot 2021-04-10 at 22 52 24

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx [hunk 1]
```diff
@@ -401,18 +409,24 @@ export default function CustomizedStepper() {
                                                 </Grid>
                                                 <Box ml={1} mb={1}>
                                                     <Grid item>
-                                                        <Typography variant='h7'>
-                                                            <FormattedMessage
-                                                                id='Apis.Details.Overview.CustomizedStepper.Endpoint'
-                                                                defaultMessage=' Endpoint'
-                                                            />
-                                                        </Typography>
+                                                        <Link
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx:412]: This `Link` is from `react-router-dom`

```js
import { Link } from 'react-router-dom';
```

Which ultimately create an anchor tag without any MUI styles, 

Please use the following pattern,  Use MUI Link and provide `react-router-dom` Link as its component

https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPI.jsx#L335-L339

**Review by tmkasun — APPROVED** [review]

**Review by tmkasun — COMMENTED** [review]
.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx (hunk 1 above)
Comment (CrowleyRajapakse) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx:412]: changed as suggested

Screenshot 2021-04-11 at 07 05 21

Screenshot 2021-04-11 at 07 05 37

Comment (tmkasun) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx:412]: great 🥳

**Pull request merged**